### PR TITLE
add slack interface

### PR DESCRIPTION
**Pull request by sent-hil**
-----Pivotal Story Info-----
Add slack interface

https://www.pivotaltracker.com/story/show/156278865
-----Pivotal Story Info-----

### CODE REVIEW

File: config.yml.sample [hunk 1]
```diff
@@ -1,6 +1,7 @@
 accessTokens:
   github: "REQUIRED"
   pivotal: "REQUIRED"
+  slack: "REQUITED"
```
Comment (dl103) [inline config.yml.sample]: requited love?

File: main.go [hunk 2]
```diff
@@ -8,17 +8,10 @@ import (
 )
 
 var (
-	GITHUB_ACCESS_TOKEN  = "GITHUB_ACCESS_TOKEN"
```
Comment (dl103) [inline main.go]: is the only access token the one stored in `config.yml.sample` now?

Comment (sent-hil) [reply main.go]: github, pivotal and slack access tokens

File: config.yml.sample (hunk 1 above)
Comment (sent-hil) [reply config.yml.sample]: 😂 freudian slip

**Pull request merged**